packages/osd-stylelint-config/config/global_selectors.json

```diff
@@ -0,0 +1,30 @@
+{
+  "#opensearch-dashboards-body": {
+    "approved": [
+      "src/core/public/rendering/_base.scss"
+    ]
+  },
+  ".app-wrapper": {
+    "approved": [
+      "src/core/public/rendering/_base.scss"
+    ]
+  },
+  "/\\.[eo]ui/": {
+    "approved": [
+      "examples/expressions_example/public/index.scss",
```

Comment:
So when new sass file gets added in the repo, do we need to add it here as well? May be we can have a script which auto-pulls this list at some point.

Reply:
When adding new Sass, we shouldn't be adding Oui overrides. This is basically just to suppress existing overrides so linting doesn't fail. If new Sass is added that overrides Oui, the linter will throw an error telling you to remove the override.

Reply:
Eventually are we going to clean up these overrides and use OUI recommended styles instead?

Reply:
Exactly. The idea is that we go through this list and figure out better ways of restyling, whether that's putting the styles in upstream OUI or creating custom classes to be put on the components, at a later point in time. This list is meant to be temporary, and ideally will be empty in the near future.

Reply:
Any follow up here @BSFishy or can we resolve this comment?